### PR TITLE
Use {{ pin_compatible('numpy') }}.

### PR DESCRIPTION
**Pull request by bdice**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bdice** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ requirements:
     - setuptools
   run:
     - python
-    - numpy >=1.10
+    - {{ pin_compatible('numpy') }}
```
Comment (isuruf) [inline recipe/meta.yaml]: This is wrong. This recipe is fine as is

**Comment by mbargull** [comment]
`pin_compatible` only makes sense if there is a package in the build/host environment that the run requirement should be compatible to.

**Comment by mbargull** [comment]
With the rerender by @mariusvniekerk, I expect https://github.com/conda-forge/garnett-feedstock/pull/1 to pass.

**Comment by bdice** [comment]
@mbargull @isuruf Thanks for the help / clarification!

**Pull request closed without merging**